inc/geometries/nablas_t.cu

```diff
@@ -0,0 +1,100 @@
+#include <iostream>
+#include <netcdf>
```

Comment:
The correct file to include is netcdf.h

Reply:
In fact it is already included by including file.h so maybe just remove that incluude